merge crvusd docs (#5)
CurveDocs/curve-docs: docs/LLAMMA/controller.md

> The controller contract is the contract the user interacts with to create a loan, repay and withdraw. It holds all user debt information. external liquidation are also done through it.

Reviewer: ```suggestion
The user interacts with the controller contract to create, repay or withdraw loans. It holds all user debt information. External liquidations are also done through it.
```

> each market has its own individual controller and amm.

Reviewer: ```suggestion
Each market has its own individual controller and AMM.
```

> this documentation utilieses the sfrxeth market to provice examples for contract info methods and functions.

Reviewer: ```suggestion
This documentation utilizes the sfrxETH market to provide examples of contract methods and functions.
```
